src/css/styles.css

```diff
@@ -181,4 +180,22 @@ h4 {
 p {
   font-weight: 200;
   font-size: 1em;
-}
```

Comment:
Does this mean that the closing tags of the p element were deleted?

Reply:
There was a blank line there that I deleted to bring the closing tag up.